1-js/01-getting-started/3-code-editors/article.md

```diff
@@ -1,46 +1,46 @@
-# Code editors
+# குறிமுறை தொகுப்பி
```

Comment:
I would prefer the English names for these such headings, they gonna play a very vital role in the developer's journey. 

```suggestion
# குறிமுறை தொகுப்பி (Code editors)
```

Reply:
Why leave English?

Reply:
The main reason is that when the developers referring some article on the web (which will be mostly in English) after skimming this section in Tamil language, it will be hard for them to relate when the hit some technical words, say "Code editors" in this case. So, keeping the corresponding English word after their Tamil translation for the hard/tough word would be a great help.